### PR TITLE
useful resources added

### PR DESCRIPTION
**Pull request by Dhanya-Abhirami**
Added links for competitive coding, presentation framework and Online chat. Minor spelling fix done.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,8 @@
 42. [**Youtube Multi Downloader**](https://youtubemultidownloader.com) - Download a complete playlist on youtube.
 43. [**Hotstar for Free**](http://kattappa.me/newlive?url=LINK_OF_HOTSTAR_VIDEO)- You can watch live match or anything else without having premium account on HotStar
 44. [**draw.io**](https://www.draw.io/) - Draw flowcharts, ER Diagrams easily.
+45. [**Hackerrank**](https://www.hackerrank.com/), [**Codechef**](https://www.codechef.com/) - Competitive Coding
```
Comment (Akarshit) [inline README.md]: These are already very well know, so I don't think there is benefit in adding these.

File: README.md [hunk 2]
```diff
@@ -90,6 +90,8 @@
...
+46. [**reveal.js**](https://revealjs.com/#/) - Make slides wich open on browser
```
Comment (Akarshit) [inline README.md]: This won't be probably used by 99% of the viewers. So I don't see much potential adding this.

**Review by Akarshit — CHANGES_REQUESTED** [review]

**Comment by Dhanya-Abhirami** [comment]
I have updated it. Please check it.

**Comment by Akarshit** [comment]
Thanks @Dhanya-Abhirami :)

**Pull request merged**